yt_dlp/extractor/nuum.py

```diff
@@ -0,0 +1,200 @@
+import functools
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    OnDemandPagedList,
+    UserNotLive,
+    int_or_none,
+    parse_iso8601,
+    str_or_none,
+    url_or_none,
+)
+from ..utils.traversal import traverse_obj
+
+
+class NuumBaseIE(InfoExtractor):
+
+    def _call_api(self, path, video_id, description, query={}):
+        response = self._download_json(
+            f'https://nuum.ru/api/v2/{path}', video_id, query=query,
+            note=f'Downloading {description} metadata',
+            errnote=f'Unable to download {description} metadata')
+        if error := response.get('error'):
+            raise ExtractorError(f'API returned error: {error!r}')
+        return response.get('result')
+
+    def _get_channel_info(self, channel_name):
+        return self._call_api(
+            'broadcasts/public', video_id=channel_name, description='channel',
+            query={
+                'with_extra': 'true',
+                'channel_name': channel_name,
+                'with_deleted': 'true',
+            })
+
+    def _parse_video_data(self, container):
+        stream = traverse_obj(container, ('media_container_streams', 0))
+        media = traverse_obj(stream, ('stream_media', 0))
+        media_url = traverse_obj(media, (
+            'media_meta', ('media_archive_url', 'media_url'), {url_or_none}), get_all=False)
+
+        return media_url, {
+            'id': str(container['media_container_id']),
+            'is_live': media.get('media_status') == 'RUNNING',
+            **traverse_obj(container, {
+                'title': ('media_container_name', {str}),
+                'description': ('media_container_description', {str}),
+                'timestamp': ('created_at', {parse_iso8601}),
+                'channel': ('media_container_channel', 'channel_name', {str}),
+                'channel_id': ('media_container_channel', 'channel_id', {str_or_none}),
+            }),
+            **traverse_obj(stream, {
+                'view_count': ('stream_total_viewers', {int_or_none}),
+                'concurrent_view_count': ('stream_current_viewers', {int_or_none}),
+            }),
```

Comment:
if removing the `stream` var:
```suggestion
            **traverse_obj(container, ('media_container_streams', 0, {
                'view_count': ('stream_total_viewers', {int_or_none}),
                'concurrent_view_count': ('stream_current_viewers', {int_or_none}),
            })),
```